systemvm/debian/opt/cloud/bin/setup/common.sh

```diff
@@ -93,6 +93,14 @@ setup_interface() {
        echo "  address $ip " >> /etc/network/interfaces
        echo "  netmask $mask" >> /etc/network/interfaces
      fi
+
+     if [ ! -z "$PRIVATEMTU" ] && [ $intf == "eth0" ]; then
+       echo "  mtu $PRIVATEMTU" >> /etc/network/interfaces
+     fi
+
+     if [ ! -z "$PUBLICMTU" ] && [ $intf == "eth2" ]; then
```

Comment:
@shwstppr 
have you tested with VPC VR ?
in VPC VR, eth1 is public interface, eth0 is the control interface

Reply:
@weizhouapache I've not tested this. @Pearl1594 or @NuxRo might have.
Will check